eng/testing/bump-chrome-version.proj

```diff
@@ -2,39 +2,46 @@
 
   <Import Project="..\..\Directory.Build.props" />
   <UsingTask AssemblyFile="$(WasmBuildTasksAssemblyPath)" TaskName="Microsoft.WebAssembly.Build.Tasks.GetChromeVersions" />
+  <PropertyGroup>
+    <ChromeVersionsPath>$(RepositoryEngineeringDir)testing\ChromeVersions.props</ChromeVersionsPath>
+  </PropertyGroup>
 
   <Target Name="UpdateChromeVersion">
     <GetChromeVersions
                 OSIdentifier="linux"
                 OSPrefix="Linux_x64"
                 Channel="$(ChromeChannel)"
                 MaxMajorVersionsToCheck="1"
-                IntermediateOutputPath="$(ArtifactsObjDir)">
+                IntermediateOutputPath="$(ArtifactsObjDir)"
+                ChromeVersionsPath="$(ChromeVersionsPath)">
```

Comment:
This kinda exposes the props file to the `*Get*ChromeVersions` task, which if we want to do then it should also do the writing, and be renamed to `UpdateChromeVersions`. And the task can emit a property for `VersionsChanged`.

A different approach would be to have the versions from the task in temporary properties. Then compare against the existing versions/urls from the props file, and then decide what to do. Essentially, moving that bit out to the targets itself.

Reply:
Taking option 1, `UpdateChromeVersions` task that is doing everything and returning info for `Message` if there was a change of not.